src/components/Editor/Editor.css

```diff
@@ -86,6 +86,11 @@ html[dir="rtl"] .editor-mount {
   right: -4px;
 }
 
+.editor.new-breakpoint {
+  width: 29px;
+  position: absolute;
+}
```

Comment:
we can't assume it's 29px. it's possible that the gutter will be wider because the file has 3/4/5 digit line numbers